package.json

```diff
@@ -107,12 +107,12 @@
 		"jest-environment-jsdom": "27.0.6",
 		"jest-serializer-vue": "^2.0.2",
 		"jest-transform-stub": "^2.0.0",
-		"node-sass": "^5.0.0",
+		"node-sass": "^6.0.1",
 		"raw-loader": "^4.0.1",
 		"regenerator-runtime": "^0.13.7",
 		"resolve-url-loader": "^3.1.2",
 		"sanitize-filename": "^1.6.3",
-		"sass-loader": "^10.0.5",
+		"sass-loader": "^10.2.0",
```

Comment:
the trick is to have a newer loader but not too new, otherwise it pulls in webpack v5 and :boom: